### DISCUSSION TITLE
[Support]: 0.14.0-beta2 requires port 80 and 443?

### DISCUSSION THREAD

**Discussion by luisiam** [blakeblackshear/frigate#11747] [General Support] [answered]
### Describe the problem you are having

Does beta 2 require port 80 and 443? I am using frigate in docker which I have another nginx proxy manager docker continuer which is using port 80 and 443. Beta 1 starts without issue while beta 2 won't start and throwing out the following errors in log.

### Version

0.14.0-beta2

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
2024-06-04 09:37:00.392723743  2024/06/04 09:37:00 [emerg] 850#850: bind() to [::]:80 failed (98: Address already in use)
2024-06-04 09:37:00.392727737  nginx: [emerg] bind() to [::]:80 failed (98: Address already in use)
2024-06-04 09:37:00.392729310  2024/06/04 09:37:00 [emerg] 850#850: bind() to [::]:443 failed (98: Address already in use)
2024-06-04 09:37:00.392730417  nginx: [emerg] bind() to [::]:443 failed (98: Address already in use)
```

### FFprobe output from your camera

```shell
N/A
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

M.2

### Network connection

Wired

### Camera make and model

N/A

**Comment by NickM-27** [comment, marked as answer]
in beta2 443 was required but this has changed after the beta. In any case you should not be running frigate in host mode which would make this not an issue

**Reply by luisiam** [reply]
Not using host mode solves the issue.

**Reply by goldserve** [reply]
I wish nginx ports were configurable because I do need host mode because of go2rtc HomeKit camera support.

**Reply by luisiam** [reply]
It will also be great if we can manage go2rtc in a different container while retaining the live streaming capability within frigate
